content/tutorials/xrp-ledger-businesses/list-your-exchange-on-xrp-charts.md

```diff
@@ -0,0 +1,197 @@
+| `type`      | String           | _(Optional)_ Type of trade. For example, valid values can include `buy` and `sell`. |
+| `size`      | String or Number | _(Optional)_ Amount of counter currency traded. |
+| `count`     | Integer          | _(Optional)_ Number of trade objects returned in the response. |
+
+
+### Examples
+
+***TODO: examples below okay - esp use of `limit` in URL and `count` in response message?***
+
+#### Example Request
+
+```
+GET https://api.example.com/v1/trades?market=xrp_btc&last_tid=75208825&limit=500
+```
+
+#### Example Response
+
+```json
+[
+   {
+      "tid":75209326,
+      "type":"buy",
+      "price":"0.57201",
+      "amount":"4954.0744",
+      "size":"2833.7801",
+      "timestamp":"2018-10-01T12:35:11.000Z"
+   },
+   ...
+   {
+      "tid":75208826,
+      "type":"sell",
+      "price":"0.57201",
+      "amount":"4954.0744",
+      "size":"2833.7801",
+      "timestamp":"2018-10-01T12:31:16.000Z"
+   }
+   "count":"500"
+]
+```
+
+
+
+## Provide a Current Order Book Endpoint
+
+Provide a RESTful API endpoint that returns data about the current order book in a particular market.
+
+XRP Charts will query this endpoint about every 30 seconds. ***TODO: does this frequency vary based on data retrieved?***
+
+
+### Request Format
+
+Provide a request format like the following:
+
+```
+GET {api_base_url}/v1/orderbook
+```
+
+#### Authentication
+
+We prefer to work with a publicly accessible endpoint.
+
+#### Parameter
+
+Provide the following parameter. The parameter field name is an example. You can use another name.
+
+| `Field`  | Type   | Description                                              |
+|:---------|:-------|:---------------------------------------------------------|
+| `market` | String | Returns the current order book in which XRP is either the base or counter currency. Use a suggested format of `<basecurrency>_<countercurrency>` to provide valid values like `xrp_btc`, `btc_xrp`, `xrp_usd`, or `usd_xrp`, for example. |
+
+
+### Response Format
+
+A successful response must be a JSON object that includes a timestamp and arrays of current bids and asks. The response does not need to provide the entire order book, but rather just enough data to provide a good idea of the current liquidity available in the market. The parameter field names are examples. You can use other names.
+
+| Field       | Type             | Description                                 |
+|:------------|:-----------------|:--------------------------------------------|
+| `timestamp` | String           | Time at which the response was sent in [ISO 8601](https://www.iso.org/iso-8601-date-and-time-format.html) or [Unix time](https://en.wikipedia.org/wiki/Unix_time) format. ***TODO: just making sure - either format okay?*** |
+| `bids`      | Array of objects | Bids in the order book. Each object in the array must provide the offered `price` and `amount` of XRP available at that price. |
```

Comment:
Always XRP, or should it be the base currency? (Or maybe the counter currency?)

Reply:
XRP can be either base or counter, but if its counter we will normalize it to base - I suggest just removing 'of XRP' so it says "offered `price` and `amount` available at that price"

Reply:
@mDuo13 Thank you for this great catch! @shekenahglory Thank you for clarifying. Fixing now.